schema.graphql

```diff
@@ -343,14 +343,34 @@ type MinterFilter @entity {
   "Unique identifier made up of minter filter contract address"
   id: ID!
 
-  "Associated core contract"
-  coreContract: Contract!
+  "Minters allowlisted globally on this MinterFilter"
+  minterGlobalAllowlist: [Minter!]!
 
-  "Minters allowlisted on MinterFilter"
-  minterAllowlist: [Minter!]!
+  "Minters allowlisted at a contract-level on this MinterFilter, only used for MinterFilterV2+"
```

Comment:
Also tiny nit: I'd like this comment to describe more specifically that this allowlist extends the global allowlist (minter must exist on global OR contract allowlist, and not expected to exist in both, right?).

Reply:
I've updated this to indicate that this enables a core contract to extend the global minter allowlist to allow minters that are not globally allowlisted.
(in general, not expected that a minter would be on this list and the global list, but nothing explicitly prevents that)